### PR TITLE
[ja] add Japanese translation for "Debugging DNS Resolution" page.

### PR DESCRIPTION
**Pull request by skmkzyk**
Let me add Japanese translation for "Debugging DNS Resolution" page.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6272530d7e1bb70009fa3e20#L4050).
> content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:21:1": failed to render shortcode "codenew": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/codenew.html:14:16": execute of template failed: template: shortcodes/codenew.html:14:16: executing "shortcodes/codenew.html" at <readFile $filename>: error calling readFile: stat /opt/build/repo/**content/ja/examples/admin/dns/dnsutils.yaml: no such file or directory**

Suggestion: copy `content/en/examples/admin/dns/dnsutils.yaml` to `content/ja/examples/admin/dns/dnsutils.yaml`,
and translate it into Japanese if needed.

@skmkzyk PTAL!

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+CoreDNSとkube-dnsのどちらのDeploymentでもServiceの名前は`kube-dns`です。
+{{< /note >}}
+
+
+
+Serviceを作成した場合、またはデフォルトで作成する必要があるのに表示されない場合は、[debugging Services](/docs/tasks/debug/debug-application/debug-service/)を参照してください。
+
+### DNSエンドポイントが公開されているか確認する {#are-dns-endpoints-exposed?}
+
+DNS エンドポイントが公開されていることを確認するには、`kubectl get endpoints`コマンドを使用します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:173]:
```suggestion
DNSエンドポイントが公開されていることを確認するには、`kubectl get endpoints`コマンドを使用します。
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+```
+NAME       ENDPOINTS                       AGE
+kube-dns   10.180.3.17:53,10.180.3.17:53    1h
+```
+
+エンドポイントが表示されない場合は、[debugging Services](/docs/tasks/debug/debug-application/debug-service/)ドキュメントのエンドポイントセクションを参照してください。
+
+その他のKubernetes DNSの例については、Kubernetes GitHubリポジトリの[cluster-dns example](https://github.com/kubernetes/examples/tree/master/staging/cluster-dns)を参照してください。
+
+### DNSクエリは受信/処理されているか確認する {#are-dns-queries-being-received/processed?}
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:187]:
```suggestion
### DNSクエリーは受信/処理されているか確認する {#are-dns-queries-being-received/processed?}
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+クエリがCoreDNSによって受信されているかどうかを確認するには、CoreDNS構成(別名 Corefile)に`log`プラグインを追加します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:189]:
```suggestion
クエリーがCoreDNSによって受信されているかどうかを確認するには、CoreDNS構成(別名 Corefile)に`log`プラグインを追加します。
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+```
+
+変更を保存した後、Kubernetesがこれらの変更をCoreDNS Podに伝達するまでに最大1から2分かかる場合があります。
+
+次に、いくつかのクエリを実行し、このドキュメントの上記のセクションごとにログを表示します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:227]:
```suggestion
次に、いくつかのクエリーを実行し、このドキュメントの上記のセクションごとにログを表示します。
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+CoreDNS Podがクエリを受信している場合は、ログにそれらが表示されます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:228]:
```suggestion
CoreDNS Podがクエリーを受信している場合は、ログにそれらが表示されます。
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+
+ログ内のクエリの例を次に示します:
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:230]:
```suggestion
ログ内のクエリーの例を次に示します:
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+CoreDNS-1.2.0
+linux/amd64, go1.10.3, 2e322f6
+2018/09/07 15:29:04 [INFO] plugin/reload: Running configuration MD5 = 162475cdf272d8aa601e6fe67a6ad42f
+2018/09/07 15:29:04 [INFO] Reloading complete
+172.17.0.18:41675 - [07/Sep/2018:15:29:11 +0000] 59925 "A IN kubernetes.default.svc.cluster.local. udp 54 false 512" NOERROR qr,aa,rd,ra 106 0.000066649s
+```
+
+### Serviceに適した名前空間にいるかを確認する {#are-you-in-the-right-namespace-for-the-service?}
+
+名前空間を指定しないDNSクエリは、Podの名前空間に制限されます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:245]:
```suggestion
名前空間を指定しないDNSクエリーは、Podの名前空間に制限されます。
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+
+PodとServiceの名前空間が異なる場合、DNSクエリにはServiceの名前空間を含める必要があります。
+
+このクエリは、Podの名前空間に限定されます:
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:249]:
```suggestion
このクエリーは、Podの名前空間に限定されます:
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+```shell
+kubectl exec -i -t dnsutils -- nslookup <service-name>
+```
+
+このクエリは名前空間を指定します:
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:254]:
```suggestion
このクエリーは名前空間を指定します:
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 10]
```diff
@@ -0,0 +1,291 @@
...
+```shell
+kubectl exec -i -t dnsutils -- nslookup <service-name>.<namespace>
+```
+
+名前解決の詳細については、[ServiceとPodに対するDNS](/ja/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names)を参照してください。
+
+## 既知の問題 {#known-issues}
+
+一部のLinuxディストリビューション(Ubuntuなど)は、既定でローカルDNSリゾルバ(`systemd-resolved`)を使用します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:263]:
```suggestion
一部のLinuxディストリビューション(Ubuntuなど)は、既定でローカルDNSリゾルバー(`systemd-resolved`)を使用します。
```

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 11]
```diff
@@ -0,0 +1,291 @@
...
+`systemd-resolved`は、`/etc/resolv.conf`をスタブファイルで移動して置き換え、アップストリームサーバで名前を解決するときに致命的な転送ループを引き起こす可能性があります。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:264]:
```suggestion
`systemd-resolved`は、`/etc/resolv.conf`をスタブファイルで移動して置き換え、アップストリームサーバーで名前を解決するときに致命的な転送ループを引き起こす可能性があります。
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

**Comment by nasa9084** [comment]
/assign
/cc @bells17 @riita10069 

Can you file an issue for this PR?

File: content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 12]
```diff
@@ -0,0 +1,291 @@
...
+{{< include "task-tutorial-prereqs.md" >}}  
+クラスターは、CoreDNS {{< glossary_tooltip text="addon" term_id="addons" >}}またはその前身であるkube-dnsを使用するように構成する必要があります。
+
+{{% version-check %}}
+
+<!-- steps -->
+
+### テスト環境として使用するシンプルなPodを作成する {#create-a-simple-pod-to-use-as-a-test-environment}
+
+{{< codenew file="admin/dns/dnsutils.yaml" >}}
```
Comment (Sea-n) [inline content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md:21]: Please copy `content/en/examples/admin/dns/dnsutils.yaml` to `content/ja/`, it caused build failure.

**Review by Sea-n — COMMENTED** [review]
Please refer to [Netlify build log](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6272530d7e1bb70009fa3e20#L4050), feel free to `/unhold` yourself when the issue is solved.

/hold

**Comment by Sea-n** [comment]
Hey @skmkzyk,

Do you need help with fixing build error and addressing suggested changes? Please contact us on `#sig-docs` channel of [Kubernetes Slack](https://slack.k8s.io).

Feel free to `/reopen` this PR when you are ready to move forward.
/close

**Pull request closed without merging**